src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalDays = days * 40;
```

Comment:
```suggestion
  const totalCost = days * 40;
```
this looks like you are counting days, not a cost

Reply:
agree, change variable name